config/default-disco.js

```diff
@@ -82,4 +82,9 @@ module.exports = {
 
   // https://sentry.prod.mozaws.net/operations/addons-frontend-disco-prod/
   publicSentryDsn: 'https://b9e70d0dca144344a7a5674c29b08355@sentry.prod.mozaws.net/186',
+
+  // Hybrid Content Telemetry: On by default for the Disco Pane.
+  // Note: Further configuration is required to enable collection locally or on -dev or stage.
```

Comment:
```suggestion
  // Note: Further configuration is required to enable collection locally or on -dev or -stage.
```

🤓

Reply:
I skipped this to not hold Bob up - are we consistently using `-stage` elsewhere? I thought historically we only used -dev as a way of showing a difference between that and local dev?

Reply:
I took the habit of naming the hosted envs: -dev, -stage, and -prod.